api/spec/v1.yaml

```diff
@@ -961,6 +961,17 @@ paths:
             A filter on the evm log signatures.
             Note: The filter will only match on parsed (verified) EVM events.
           example: '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
+        - in: query
+          name: evm_log_contract_addr_eth
```

Comment:
I'd drop the `_eth` suffix. I checked and we never use it (or the `eth_` prefix) for _query_ params. I think it's explicit enough if the docstring says we want the hex-encoded eth address. Also, if we ever want to support oasis-style addresses, I expect we'll follow the lead of a few other [similar params](https://github.com/oasisprotocol/nexus/blob/f495dac742ed91fd6e32336dfbc99d00955c9abd/api/spec/v1.yaml#L883-L895) so far, where we accept either format.

I'd _also_ drop `evm_log`, because that's true now but might not be in the future (Cipher or similar).

How do you like `emitting_contract_addr` to make it clearer how this contract is related? Or just `contract_add`.

Reply:
> we never use [_eth suffix] (or the eth_ prefix)

![image](https://github.com/oasisprotocol/nexus/assets/38229038/5851b2c2-831c-4f30-b7e5-c83e8d68781a)

Reply:
I think what I'll do is take oasis format address like everything else and go through address_preimages in the db :skull:

Reply:
I thought about that option too. It's fairly contrived because now the frontend will have to convert from eth to bech32, only for us to convert (somewhat expensively) back. If you feel like supporting both formats (bech32 and eth-hex), I'm in favor of it; it's the most consistent with analogous params like `tx_hash`.